src/lbp.jl

```diff
@@ -0,0 +1,144 @@
+type UNIFORM_PARAMS
+	table::Dict{BitArray{1}, Int}
+	count::Int
+	non_uniform_pattern::BitArray{1}
+end
```

Comment:
@timholy Does ```BitArray{1}``` need to be passed as a parameter to make it efficient or is this ok?